docs/read_the_docs/source/advanced_tutorial/writing_housing_characteristics.rst

```diff
@@ -7,4 +7,5 @@ Guidelines
 =============
 
 1. All lines have the line ending characters '\\r\\n' (i.e., "crlf" or "carriage return, line feed").
-2. All non-dependency columns have format '%.6f'.
\ No newline at end of file
+2. All non-dependency columns have format '%.6f'.
+3. Comment lines at the end of a housing characteristics file, indicated by the "#" symbol, can be added.
```

Comment:
The implementation appears to handle comment lines anywhere in the file, not just at the end of the file, right?

Reply:
Good point. I didn't include the necessary lines of code for header comments, and comments inserted in the middle of the tsv.
- Header comments: looks like I need to update get_file_data(...)
- Comments inserted in the middle of the tsv: looks like I need to update get_option_name_from_sample_number(...)